core/templates/pages/exploration-editor-page/translation-tab/voiceover-card/voiceover-card.component.ts

```diff
@@ -174,7 +183,6 @@ export class VoiceoverCardComponent implements OnInit {
   }
 
   updateActiveContent(): void {
-    this.voiceoversAreLoaded = false;
```

Comment:
It is not required here, since this variable is instantiated at the ngOnInit method.